src/styles/blocks/calendar.scss

```
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $calendar-padding;
+  gap: $calendar-gap;
+  width: getCalendarWidth(
+    $calendar__day-size,
+    $calendar-gap,
+    $calendar-padding
+  );
+  margin: 0 auto;
+
+  @each $days, $index in
+    (mon: 0, tue: 1, wed: 2, thu: 3, fri: 4, sat: 5, sun: 6) {
```

```suggestion
    (mon: 0, tue: 1, wed: 2, thu: 3, fri: 4, sat: 5, sun: 6) {
```
you can move it to the variable